### ISSUE TITLE
Crash New Game

### ISSUE THREAD

**Issue by HSteven1406** [TitansBane/Elysium#94]
Hello,

Each time I launch a new game, I have a CTD.

**Comment by SaitoAsuka98** [comment]
yes, i have the same problem, but i've been looking for every solution, and i still stuck when i click new game. have u find the solution yet ? i already try to re-installing the mod and the game. but nothing change

**Comment by HSteven1406** [comment]
I uninstalled and change for another wabbajack... So, I can't help
